oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/worker/MetricsPersistentWorker.java

```diff
@@ -95,6 +100,8 @@
         this.sessionTimeout = storageSessionTimeout;
         this.persistentCounter = 0;
         this.persistentMod = 1;
+        // Convert TTL to millis to make further calculation easier.
+        this.metricsDataTTLInMillis = metricsDataTTL * 24 * 60 * 60 * 1000;
```

Comment:
Consider using `TimeUnit.DAYS.toMilliSeconds` for better readability.

Reply:
Where do we have this method?

Reply:
It’s in JDK `TimeUnit.DAYS.toMillis`


https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/TimeUnit.html#toMillis-long-

Reply:
Done